### DISCUSSION TITLE
studio for manage data a vercel postgres ?

### DISCUSSION THREAD

**Discussion by yasseralsaidi** [vercel/next.js#54179] [Help]
### Summary

how can i manage table data a vercel postgres (like import / edit / delete...) is there any studio?

**Comment by Nadeera3784** [comment]
Hi @yasseralsaidi,
I guess you are talking about a graphical database management tool.
i personally  use tableplus.
in tablePlus you need to create a new connection with  postgres as type. so copy your credentials from settings and put those in tablePlus.

![image](https://github.com/vercel/next.js/assets/22231476/b7a2126c-af8b-4dfb-83d7-a635e2d35515)

**Reply by yasseralsaidi** [reply]
@Nadeera3784 thank you for your suggestion, i tried it but it still need to install it and buy the licence to use it
i'm looking for vercel  own graphical database management tool as you said, but i think they don't have it yet!

**Reply by Nadeera3784** [reply]
@yasseralsaidi  Yes you are correct. vercel does not have it yet. so you can use alternative solutions like,

https://dbeaver.io/
https://www.pgadmin.org/

**Comment by yasseralsaidi** [comment]
i see, thank you again @Nadeera3784 🤝🏻
